src/folding/nova/mod.rs

```diff
@@ -0,0 +1,62 @@
+use ark_crypto_primitives::sponge::Absorb;
+use ark_ec::{CurveGroup, Group};
+use ark_std::fmt::Debug;
+use ark_std::{One, Zero};
+
+use crate::pedersen::{Params as PedersenParams, Pedersen};
+
+pub mod nifs;
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct CommittedInstance<C: CurveGroup> {
+    pub cmE: C,
+    pub u: C::ScalarField,
+    pub cmW: C,
+    pub x: Vec<C::ScalarField>,
+}
+impl<C: CurveGroup> CommittedInstance<C> {
+    pub fn empty() -> Self {
+        CommittedInstance {
+            cmE: C::generator(),
```

Comment:
`C::zero` seems closer to `empty`

Reply:
Absolutely! Thanks for pointing it out ^^, updated